app/controllers/hub/notes_controller.rb

```diff
@@ -24,5 +25,10 @@ def create
     def note_params
       params.require(:note).permit(:body).merge(user: current_user, client: @client)
     end
+
+    def create_notifications
+      mentioned_user_ids = params.dig(:note, :mentioned_ids).split(",")
+      mentioned_user_ids.each { |id| UserNotification.create(notifiable: @note, user_id: id) }
```

Comment:
I noticed in the Notes model it doesn't have the same association that tax_return_assignment has to notifiable with `has_one :user_notification, as: :notifiable`

Reply:
@embarnard Are we using that association anywhere? I think here it might have to be a has_many (since many users can be tagged in the same note). I'm going to try to get away without it for now and then if we end up needing it, I'll add it in.